src/platforms/javascript/common/troubleshooting/index.mdx

```diff
@@ -66,9 +66,19 @@ If `instrument.js` displays in your console while debugging, add Sentry to your
 
 ## Dealing with Ad-Blockers
 
+When you are using our CDN, ad-blocking or script-blocking extensions may prevent our SDK from being fetched and initialized properly. Because of this, any call to SDKs API will fail and may cause your application to misbehave.
+
+Additionally, even when the SDK is downloaded and initialized correctly, Sentry endpoints that need to receive captured data may be blocked as well. This will prevent any error reports, sessions health or performance data to be delivered, and effectively not available in your Sentry dashboard.
```

Comment:
I think you're trying to say this data won't be available to you in the Sentry app UI. If so then, how I've updated it below is the way we reference that.

```suggestion
Additionally, even when the SDK is downloaded and initialized correctly, Sentry endpoints that need to receive captured data may be blocked as well. This prevents any error reports, sessions health, or performance data from being delivered, making it effectively unavailable in [sentry.io](https://sentry.io).
```